.pipelines/templates/release.yml

```diff
@@ -5,64 +5,81 @@ pr: none
+            tag: $(version)
 
-- job: githubRelease
-  displayName: GitHub Release
-  dependsOn: version
-  variables:
-    version: $[ dependencies.Version.outputs['setVer.version'] ]
-  steps:
-    - task: GitHubRelease@0
-      inputs:
-        gitHubConnection: $(GITHUB_CONNECTION)
-        tagSource: manual
-        tag: $(version)
+    - job: PublishToPyPi
+      displayName: Publish to PyPi
+      dependsOn: version
+      strategy:
+        matrix:
+          Analyzer:
+            WORKING_FOLDER: 'presidio-analyzer'
+          Anonymizer:
+            WORKING_FOLDER: 'presidio-anonymizer'
+      steps:
+        - template: ./publish-to-pypi.yml
+          parameters:
+            WORKING_FOLDER: '$(WORKING_FOLDER)'
 
-- job: PublishToPyPi
-  displayName: Publish to PyPi
-  dependsOn: version
-  strategy:
-    matrix:
-      Analyzer:
-        WORKING_FOLDER: 'presidio-analyzer'
-      Anonymizer:
-        WORKING_FOLDER: 'presidio-anonymizer'
-  steps:
-    - template: ./publish-to-pypi.yml
-      parameters:
-        WORKING_FOLDER: '$(WORKING_FOLDER)'
-
-- job: BuildAndPushContainers
-  displayName: Build and Push Containers
-  dependsOn: version
-  pool:
-    vmImage: 'ubuntu-16.04'
-  variables:
-    version: $[ dependencies.Version.outputs['setVer.version'] ]
-    REGISTRY_NAME: '$(ACR_REGISTRY_NAME).azurecr.io/'
-    IMAGE_PREFIX: "public/"
-  steps:
-    - template: ./build-and-push-containers.yml
-      parameters:
-        REGISTRY_NAME: $(REGISTRY_NAME)
-        IMAGE_PREFIX: $(IMAGE_PREFIX)
-        TAG: ":$(version)"
-        AZURE_SUBSCRIPTION: $(ACR_AZURE_SUBSCRIPTION)
-    - template: ./build-and-push-containers.yml
-      parameters:
-        REGISTRY_NAME: $(REGISTRY_NAME)
-        IMAGE_PREFIX: $(IMAGE_PREFIX)
-        TAG: ":v2-latest"
-        AZURE_SUBSCRIPTION: $(ACR_AZURE_SUBSCRIPTION)
+    - job: BuildAndPushContainers
+      displayName: Build and Push Containers
+      dependsOn: version
+      pool:
+        vmImage: 'ubuntu-16.04'
+      variables:
+        version: $[ dependencies.Version.outputs['setVer.version'] ]
+        REGISTRY_NAME: '$(ACR_REGISTRY_NAME).azurecr.io/'
+        IMAGE_PREFIX: "public/"
+      steps:
+        - template: ./build-and-push-containers.yml
+          parameters:
+            REGISTRY_NAME: $(REGISTRY_NAME)
+            IMAGE_PREFIX: $(IMAGE_PREFIX)
+            TAG: ":$(version)"
+            AZURE_SUBSCRIPTION: $(ACR_AZURE_SUBSCRIPTION)
+        - template: ./build-and-push-containers.yml
+          parameters:
+            REGISTRY_NAME: $(REGISTRY_NAME)
+            IMAGE_PREFIX: $(IMAGE_PREFIX)
+            TAG: ":v2-latest"
+            AZURE_SUBSCRIPTION: $(ACR_AZURE_SUBSCRIPTION)
+  - stage: DeployProd
+    dependsOn: Release
+    variables:
+      REGISTRY_NAME: 'mcr.microsoft.com/'
+      VERSION: $[ stageDependencies.Release.version.outputs['setVer.version'] ]
+    displayName: Deploy to Production environment
+    jobs:
+      - template: ./deploy-stage.yml
```

Comment:
I would just rename this to plain deploy

Reply:
done